### PR TITLE
Login and register

### PR DESCRIPTION
**Pull request by ofirtzabari**
create login and register pages using firebase

### CODE REVIEW

**Review by Omer-Levi — APPROVED** [review]
good job

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -9,11 +9,17 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
-        android:supportsRtl="true"
+        android:supportsRtl="false"
```
Comment (Shalevro2) [inline app/src/main/AndroidManifest.xml:12]: Our App language is English?

**Review by Shalevro2 — APPROVED** [review]
LGTM

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (ofirtzabari) [reply app/src/main/AndroidManifest.xml:12]: i dont think its matter
its only change the position of objects

**Pull request merged**